lib/c_glib/src/thrift/c_glib/transport/thrift_socket.h

```diff
@@ -50,11 +50,8 @@ struct _ThriftSocket
 
   /* private */
   gchar *hostname;
-  gshort port;
+  guint port;
   int sd;
-  guint8 *buf;
```

Comment:
"buf" is not used anywhere in thrift_socket.c and it was marked as private.  This means it was there for no reason that's useful.